applications/NXmx.nxdl.xml

```diff
@@ -652,11 +744,6 @@
 						<doc>
 							The wavelength spread FWHM for the corresponding
 							wavelength(s) in incident_wavelength.
-
```

Comment:
Likewise, could this note be preserved?  Added in #717 as part of handling per-shot wavelengths.

Reply:
Fixed in commit  1fe10e2